azurerm/helpers/azure/batch_pool_test.go

```diff
@@ -0,0 +1,138 @@
+package azure
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/Azure/azure-sdk-for-go/services/batch/mgmt/2018-12-01/batch"
+)
+
+func TestExpandBatchPoolNetworkConfiguration(t *testing.T) {
```

Comment:
We typically don't write tests for these as the acc tests will catch any issues

Reply:
removed